realsense2_camera/include/gl_window.h

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 Intel Corporation. All Rights Reserved.
```

Comment:
2024

Reply:
CI is failing if we update it to 2024. We should update the script [pr_check.sh](https://github.com/IntelRealSense/realsense-ros/blob/ros2-development/scripts/pr_check.sh#L46) which checks the year first and update in all of the source files.

Reply:
Decided to do that in different PR.